contributing-device-support.md

```diff
@@ -1,120 +1,207 @@
-# Contributing new board support to resin.io
+# Contributing New Board Support to Resin.io
 
-Pre-requisites: yocto BSP layer for your particular board, supplying the BSP (board support package). Should be compatible to the yocto releases resin currently supports.
+Pre-requisites: a [Yocto](https://www.yoctoproject.org) Board Support Package (BSP) layer for your particular board. It should be compatible to the Yocto releases resinOS supports.
 
-Every repository used to build a resin.io host Operating System is typically named resin-`<board-family>`. For example, consider the [resin-intel][resin-intel repo] repository which can be used to build a resin.io image for
-the Intel NUC boards.
+Repositories used to build resinOS host Operating System (OS) are typically named resin-`<board-family>`. For example, consider [resin-raspberrypi](https://github.com/resin-os/resin-raspberrypi) which is used for building the OS for [Raspberryi Pi](https://raspberrypi.org), or [resin-intel][resin-intel repo] repository which can be used to build a resin.io image for the Intel NUC boards.
 
-### Repository breakout
------------------------
+Contributing support for a new board consist of creating a a Yocto package that includes:
```

Comment:
should be:

Contributing support for a new board consists of creating a a Yocto package that includes: